### PR TITLE
Generating Supplementary Figure from Initial Z Parameter Sweep

### PR DESCRIPTION
**Pull request by gwaygenomics**
Here I add logic to the first module (`1.initial-z-sweep`) to plot a summary of the parameter sweep effort in a single figure. This figure will be supplementary in a future paper:

z_dimension_sweep_summary

I am also rerunning the module, adding all updated figures, and adding `cowplot` to the environment file.

### CODE REVIEW

**Comment by gwaygenomics** [comment]
> Is there a reason that there is a different number of learning rates used for TARGET vs TCGA and 
GTeX?

I didn't try as many combinations - takes a while to search through GTEX. I also found that many parameter combinations worked reasonably well.

Also, in panel A, I removed hyperparameter combinations that failed to converge. This is why there are fewer points.

> I assume the loss is reconstruction loss? Or is this total loss?

Good point. Both are total loss, but the VAE includes KL. I will need to be clear in a figure legend.
